docs/index.md

```diff
@@ -97,4 +95,5 @@ developers/roles.md
 :caption: Maintainers
 
 maintainers/develop.md
+maintainers/release.md
```

Comment:
did you forget to add this to the commit?

Reply:
yep, fixed!